### ISSUE TITLE
Sparkline Exception

### ISSUE THREAD

**Issue by roomtek** [appreciated/apexcharts#10]
I am getting this exception:

com.fasterxml.jackson.databind.exc.InvalidDefinitionException: No serializer found for class com.github.appreciated.apexcharts.config.chart.Sparkline and no properties discovered to create BeanSerializer (to avoid exception, disable SerializationFeature.FAIL_ON_EMPTY_BEANS) (through reference chain: com.github.appreciated.apexcharts.config.Chart["sparkline"])

when using 

```
ApexCharts barChart = new ApexCharts()
				.withChart(ChartBuilder.get().withSparkline(new Sparkline()).withStacked(true).build())
```

**Comment by appreciated** [comment]
@roomtek this usually happens when the getter and setters are missing.

**Comment by appreciated** [comment]
fixed by 8be0b3c4b90b75ec8389a08cc8cb8cc8ab91ede6 and available in [Version 0.3.2](https://vaadin.com/directory/component/apexchartsjs/0.3.2)
